pandas/core/frame.py

```diff
@@ -4922,10 +4932,10 @@ def f(vals):
 
     # ----------------------------------------------------------------------
     # Sorting
-
+    # TODO: Just move the sort_values doc here.
```

Comment:
is this still needed?

Reply:
Yeah. This will be solved when the docstring NDFrame.sort_values is just moved to DataFrame.sort_values.